src/main/scala/com/itechart/ny_accidents/parse/PopulationParser.scala

```diff
@@ -15,9 +14,9 @@ import scala.util.control.Exception
 
 @Singleton
 class PopulationParser {
-  val injector: Injector = Guice.createInjector(new GuiceModule)
-  val districtsService: DistrictsService = injector.getInstance(classOf[DistrictsService])
-  val districtsStorage: DistrictsStorage = injector.getInstance(classOf[DistrictsStorage])
+  lazy val injector = Injector.injector
```

Comment:
I guess you can import Injector.injector so you don't need a separate field to store ref to it